src/test/kotlin/no/nav/syfo/oppfolgingstilfelle/bit/OppfolgingstilfelleBitSpek.kt

```diff
@@ -332,6 +332,162 @@ class OppfolgingstilfelleBitSpek : Spek({
+                    fom = LocalDate.now().minusDays(6),
+                    tom = LocalDate.now(),
+                    virksomhetsnummer = "987654330",
+                ),
+            )
+
+            val oppfolgingstilfelleList = oppfolgingstilfelleBitList.generateOppfolgingstilfelleList()
+            oppfolgingstilfelleList.size shouldBeEqualTo 1
+            val oppfolgingstilfelle = oppfolgingstilfelleList.first()
+
+            val tilfelleDuration = Period.between(
+                oppfolgingstilfelle.start,
+                oppfolgingstilfelle.end,
+            ).days
+            tilfelleDuration shouldBeEqualTo 16
+            oppfolgingstilfelle.virksomhetsnummerList.size shouldBeEqualTo 0
+            oppfolgingstilfelle.arbeidstakerAtTilfelleEnd shouldBeEqualTo false
+        }
+
+        it("should return 1 Oppfolgingstilfelle and arbeidstaker when sykmelding-ny, sykmelding-sendt and sykmelding-bekreftet") {
+            val oppfolgingstilfelleBitList = listOf(
+                defaultBit.copy(
+                    createdAt = nowUTC(),
+                    inntruffet = nowUTC(),
+                    tagList = listOf(SYKMELDING, BEKREFTET, PERIODE, INGEN_AKTIVITET),
+                    fom = LocalDate.now().minusDays(16),
+                    tom = LocalDate.now().minusDays(8),
+                    virksomhetsnummer = null,
+                ),
+                defaultBit.copy(
+                    createdAt = nowUTC(),
+                    inntruffet = nowUTC(),
+                    tagList = listOf(SYKMELDING, SENDT, PERIODE, INGEN_AKTIVITET),
+                    fom = LocalDate.now().minusDays(6),
+                    tom = LocalDate.now().minusDays(4),
+                    virksomhetsnummer = "987654330",
+                ),
+                defaultBit.copy(
+                    createdAt = nowUTC(),
+                    inntruffet = nowUTC(),
+                    tagList = listOf(SYKMELDING, NY, PERIODE, INGEN_AKTIVITET),
+                    fom = LocalDate.now().minusDays(3),
+                    tom = LocalDate.now(),
+                    virksomhetsnummer = "987654330",
+                ),
+            )
+
+            val oppfolgingstilfelleList = oppfolgingstilfelleBitList.generateOppfolgingstilfelleList()
+            oppfolgingstilfelleList.size shouldBeEqualTo 1
+            val oppfolgingstilfelle = oppfolgingstilfelleList.first()
+
+            val tilfelleDuration = Period.between(
+                oppfolgingstilfelle.start,
+                oppfolgingstilfelle.end,
+            ).days
+            tilfelleDuration shouldBeEqualTo 16
+            oppfolgingstilfelle.virksomhetsnummerList.size shouldBeEqualTo 1
+            oppfolgingstilfelle.arbeidstakerAtTilfelleEnd shouldBeEqualTo true
+        }
+
+        it("should return 1 Oppfolgingstilfelle and arbeidstaker when sykmelding-ny, sykpengesoknad-sendt and sykmelding-bekreftet") {
```

Comment:
Hvorfor er man arbeidstaker når man har sendt en søknad?

Reply:
Fordi vi tar bare inn sykepengesøknader som har orgnummer != null: se KafkaSyketilfellebit.isRelevantForOppfolgingstilfelle()

Reply:
Åja, lurt at du la til test for det da! 🎉